README.rst

```diff
@@ -169,9 +169,8 @@ If you're having trouble, we have discussion forums at https://discuss.openedx.o
 
 Our real-time conversations are on Slack. You can request a `Slack invitation`_, then join our `community Slack workspace`_.
 
-For more information about these options, see the `Getting Help`_ page.
+For more information about these options, see the `Getting Help <https://openedx.org/getting-help>`__ page.
```

Comment:
Why change just the syntax of this URL?

Reply:
See notes in this commit message: https://github.com/openedx/edx-cookiecutters/pull/355/commits/6c3058292104383332f7f7a97d1ebb85f4176aba -- happy to have other suggestions, though.